src/docs/docs/api.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Making an API Public"
+---
+
+## Introduction
+
+In this document, we explain both how you make an API public and the considerations to take into account before doing so.
+
+## Definitions
+
+### Managed Client
+
+A managed client is one that Sentry controls, such as our frontend. In this type of client, we control both which endpoints are used and the attributes used by those endpoints.
+
+### Partially Managed Client
+
+A partially-managed client is one that we only partially control, since we do not control the version of the client that is running for our users; for example, our Sentry CLI. While we control the endpoints the CLI uses, once the CLI is installed and running in a customer's implementation, we don't control the version of the client a user runs.
+
+### Wild Client
```

Comment:
External might be a better name here.

Reply:
Yea, had my doubts about "wild". I'll change to external.